spec/unit/content_item_loader_spec.rb

```diff
@@ -0,0 +1,67 @@
+RSpec.describe ContentItemLoader do
+  include ContentStoreHelpers
+
+  let!(:item_request) { stub_content_store_has_item("/my-random-item") }
+
+  describe ".load" do
+    it "caches calls to the content store" do
+      ContentItemLoader.load("/my-random-item")
+      ContentItemLoader.load("/my-random-item")
+
+      expect(item_request).to have_been_made.once
+    end
+
+    context "with a missing content item" do
+      before do
+        stub_content_store_does_not_have_item("/my-missing-item")
+      end
+
+      it "returns (but does not raise) the original exception" do
+        expect(ContentItemLoader.load("/my-missing-item")).to be_a(GdsApi::HTTPErrorResponse)
+      end
+    end
+
+    context "With ALLOW_LOCAL_CONTENT_ITEM_OVERRIDE=true" do
+      before do
+        ENV["ALLOW_LOCAL_CONTENT_ITEM_OVERRIDE"] = "true"
+        stub_const("ContentItemLoader::LOCAL_ITEMS_PATH", "spec/fixtures/local-content-items")
+      end
+
+      after do
+        ENV["ALLOW_LOCAL_CONTENT_ITEM_OVERRIDE"] = nil
+      end
+
+      context "with a local JSON file" do
+        let!(:item_request) { stub_content_store_has_item("/my-json-item") }
+
+        it "loads from that" do
+          response = ContentItemLoader.load("/my-json-item")
+
+          expect(item_request).not_to have_been_made
+          expect(ContentItemFactory.build(response).schema_name).to eq("json_page")
+        end
+      end
+
+      context "with a local YAML file" do
+        let!(:item_request) { stub_content_store_has_item("/my-yaml-item") }
+
+        it "loads from that file" do
```

Comment:
Ditto here about the test name, but again, not a blocker.

Reply:
Ta! Fixed.